pkg/clients/ecr/repository_policy.go

```diff
@@ -209,7 +220,10 @@ func IsRepositoryPolicyUpToDate(local, remote *string) bool {
 		return false
 	}
 
-	return cmp.Equal(localUnmarshalled, remoteUnmarshalled)
+	sortSlicesOpt := cmpopts.SortSlices(func(x, y interface{}) bool {
+		return x.(string) < y.(string)
```

Comment:
Not sure we can get away with this, as anything in a slice that isn't a string will cause a panic. Every example I can find of an AWS IAM policy has all values as strings and I've not found any other documentation to suggest you can pass non-strings to a policy (even when comparing numbers, they're sent as quoted strings).

Feedback appreciated.

Reply:
We can handle the input types we know and if an unexpected type came in, returning `false` would be better than panic IMO.

Reply:
Resolved by attempting to convert `x` and `y` to strings and returning `false` if either variable is an unknown type.